app/views/users/new.html.erb

```diff
@@ -3,47 +3,47 @@
 
   <div class="signup-breadcrumb">
      <div style="background-color:#eee;">
-         Create Account
+         Create Account/Créer le Compte 
      </div>
-     <div>
-         Additional Information
+     <div style= "padding-top: 3% ; padding-bottom: 4%">
```

Comment:
It's generally a good idea to keep styles out of HTML like this because it's harder to re-use. Can you add a class to this div and move the padding stuff into `users.scss`?